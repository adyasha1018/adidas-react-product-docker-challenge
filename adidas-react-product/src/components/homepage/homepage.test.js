import React from "react";
import { shallow } from "enzyme";
import Homepage from "./homepage.component";
describe("Homepage", () => {
    const wrapper = shallow(<Homepage />)
    it("should render Homepage component", () => {
      expect(wrapper).toMatchSnapshot();
      expect(wrapper.find('div.homepage').exists()).toEqual(true);
      expect(wrapper.find('div.homepage').children().length).toEqual(2);
      expect(wrapper.find('div.homepage span.text').text()).toEqual("Welcome To Adidas World");
      expect(wrapper.find('div.homepage div.logo img').exists()).toEqual(true);
    });
});
