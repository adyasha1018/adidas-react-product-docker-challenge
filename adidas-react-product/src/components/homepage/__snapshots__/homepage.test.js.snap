// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render Homepage component 1`] = `ShallowWrapper {}`;

exports[`Homepage should render Homepage component 1`] = `ShallowWrapper {}`;

exports[`should render Homepage component 1`] = `ShallowWrapper {}`;
